packages/html-reporter/src/testCaseView.tsx

```diff
@@ -52,6 +49,21 @@ export const TestCaseView: React.FC<{
   </div>;
 };
 
+function renderAnnotationDescription(description: string) {
+  if (description.startsWith('http://') || description.startsWith('https://'))
```

Comment:
For proper URL validation:

```suggestion
  try {
    if (['http:', 'https:'].includes(new URL(description).protocol))
      ...
  } catch {}
  return description;
```